style.css

```diff
@@ -410,4 +408,223 @@ html body .section_about_me .card_container2 .li_card_container_about .tittle .t
+  color: #6b778c;
+  padding-left: 16px;
+}
+
+.section-contact-form .div_form_container .contact-form .div_field fieldset input[type="text"]:hover {
+  border: 1px solid #60c095;
+}
+
+.section-contact-form .div_form_container .contact-form .div_field fieldset input[type="text"]:focus {
+  outline: none;
+  border: 1px solid #60c095;
+}
+
+.section-contact-form .div_form_container .contact-form .div_field fieldset input[type="email"] {
+  height: 48px;
+  width: 327px;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #6b778c;
+}
+
+.section-contact-form .div_form_container .contact-form .div_field fieldset input[type="email"]:hover {
+  border: 1px solid #60c095;
+}
+
+.section-contact-form .div_form_container .contact-form .div_field fieldset input[type="email"]:focus {
+  outline: none;
+  border: 1px solid #60c095;
+}
+
+.section-contact-form .div_form_container .contact-form .div_field fieldset textarea {
+  height: 114px;
+  width: 327px;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #6b778c;
+  font-family: "Inter", sans-serif;
+  padding-top: 12px;
+}
+
+.section-contact-form .div_form_container .contact-form .div_field fieldset textarea:hover {
+  border: 1px solid #60c095;
+}
+
+.section-contact-form .div_form_container .contact-form .div_field fieldset textarea:focus {
+  outline: none;
+  border: 1px solid #60c095;
+}
+
+.section-contact-form .div_form_container .contact-form .divbtn_field {
+  margin-top: 68px;
+}
+
+.section-contact-form .div_form_container .contact-form .divbtn_field .btn-form {
+  padding: 12px;
+  margin-top: 12px;
+  gap: 10px;
+  width: auto;
+  height: auto;
+  bottom: 8px;
+  border-radius: 4px;
+  color: #fff;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  border: 0;
+  cursor: pointer;
+}
+
+.section-contact-form .contact-form .social_links {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  list-style-type: none;
+  gap: 20px;
+  padding: 0;
+  margin-top: 0;
+}
+
+.email_footer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 189px;
+  width: 375px;
+  border-radius: nullpx;
+  margin-bottom: 100px;
+  margin-left: 24px;
+}
+
+.link_email_footer {
+  text-decoration: none;
+}
+
+.email_footer .heading_footer {
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 32px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #42526e;
+}
+
+.rounded {
+  border-top: 8px solid black;
+  border-radius: 5px;
+  width: 35%;
+}
+
+.divider3 {
+  display: flex;
+  justify-content: center;
+}
+
+.btn_effects {
+  background: #36b37f;
+}
+
+.btn_effects:hover {
+  background: #36b37f;
+}
```

Comment:
- [ ] The button interaction you chose for the hover effect has a box shadow of `0px 8px 16px rgba(54, 179, 127, 0.24)` with a `border-radius` of `4px`. Kindly include these declarations so that the `hover` effect is noticeable. 👍

Reply:
Dear @Gambit142 , thanks for your feedback. Could you check my implementation?
![image](https://user-images.githubusercontent.com/65084923/207707726-3d95af23-f0a8-46d7-befc-4bb618309f15.png)

Reply:
Hi @chelobotix, you can submit it for review. Great work 👏